release notes/v0.46.0.md

```diff
@@ -89,6 +141,24 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 - [#3136](https://github.com/grafana/k6/pull/3136) Updates k6 Go dependencies.
 - [#3131](https://github.com/grafana/k6/pull/3131) Updates our Pull Request template to be more structured and include a checklist for contributors.
 - [#3177](https://github.com/grafana/k6/pull/3177) Updates the version of golangci-lint we use to the latest version.
+- [browser#849](https://github.com/grafana/xk6-browser/pull/849) Updates `nil` session log warn to debug level in page attach.
+- [browser#881](https://github.com/grafana/xk6-browser/pull/881) Adds a new `browser_http_req_failed` metric.
```

Comment:
Would this better placed under improvements?

Reply:
It is under improvements section, did you mean "enhancements" ?

Reply:
Fixed in [a7cf0a2](https://github.com/grafana/k6/pull/3260/commits/a7cf0a2710efff9f593b3f1e28e4e281df8e2a7a).